src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.LinkedList;
+import java.util.Arrays;
+import java.io.Serializable;
+import java.net.URL;
+import java.util.logging.Logger;
+import com.google.maps.GeoApiContext;
+import com.google.maps.PlacesApi;
+import com.google.maps.NearbySearchRequest;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.LocationType;
+import com.google.maps.model.Photo;
+import com.google.maps.model.PlaceType;
+import com.google.maps.model.RankBy;
+import com.google.maps.model.Geometry;
+import com.google.maps.model.PlacesSearchResponse;
+import com.google.maps.model.PlacesSearchResult;
+import java.util.logging.Logger;
+
+/** BusinessesService object representing all businesses components of the webapp **/
+public class BusinessesService {
+  
+  private List<Listing> businesses;
+  private final String KEY = "REDACTED";
+  private final static Logger LOGGER = Logger.getLogger(BusinessesService.class.getName());
+
+  /** Create a new Businesses instance
+  * @param businesses businesses from SmallCityService
+  **/
+  public BusinessesService(List<Listing> businesses) {
+    this.businesses = businesses;
+  }
+
+  public List<Listing> getBusinessesFromPlacesApi(User user) {
+    LatLng latLng = new LatLng(user.getGeolocation().lat, user.getGeolocation().lng);
+    final GeoApiContext context = new GeoApiContext.Builder()
+            .apiKey(KEY)
+            .build();
+    NearbySearchRequest request = PlacesApi.nearbySearchQuery(context, latLng);
+    try {
+      PlacesSearchResponse response = request.type(PlaceType.STORE)
+              .rankby(RankBy.DISTANCE)
+              .await();
+      final int allowedSearchRequests = 3;
```

Comment:
Perhaps make this a constant at the top of the file, and refer to it here.

Reply:
Got it